webapp/src/main/java/com/box/l10n/mojito/entity/RepositoryAIPrompt.java

```diff
@@ -0,0 +1,49 @@
+package com.box.l10n.mojito.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(
+    name = "repository_ai_prompt",
+    indexes = {
```

Comment:
Re the two indexes added - is it the case that the automation would mostly query the `repository_ai_prompt` table by either repo_id **or** prompt_type_id? 

If it's the case that we would generally query by both in a single query (i.e.: get all string check prompts for repo X - we should add a single compound index instead).

Reply:
You're correct, I'm querying by repo id + prompt type name, with the introduction of promptTypeId I think this can be changed to a non-unique compound index of all three ids instead.